framework/src/outputs/Console.C

```diff
@@ -366,7 +366,7 @@ Console::output(const ExecFlagType & type)
       _old_nonlinear_norm = std::numeric_limits<Real>::max();
 
     _console << std::setw(2) << _nonlinear_iter
-             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << '\n';
+             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << std::endl;
```

Comment:
No, I did not, but we can imagine that the number of this flush at most is hundreds, its impact on timing should be negligible. I'd rather to keep more flushes unless that flush is in places like a kernel computeresidual.